src/main/java/com/github/fppt/jedismock/operations/scripting/ScriptLoad.java

```diff
@@ -0,0 +1,48 @@
+package com.github.fppt.jedismock.operations.scripting;
+
+import com.github.fppt.jedismock.datastructures.Slice;
+import com.github.fppt.jedismock.operations.AbstractRedisOperation;
+import com.github.fppt.jedismock.operations.RedisCommand;
+import com.github.fppt.jedismock.server.Response;
+import com.github.fppt.jedismock.storage.RedisBase;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.List;
+
+@RedisCommand("scriptload")
```

Comment:
I think this won't work, as we don't have `scriptload` command. We have `script load`, which is `script` command with `load` parameter. Also there is no comparison test for `script load`